rca/static_src/sass/components/_carousel.scss

```diff
@@ -225,6 +235,31 @@
         }
     }
 
+    &__mobile-controls-group {
+        display: flex;
+        justify-content: center;
+        gap: $gutter;
+        pointer-events: auto; // ensure prev/next buttons are clickable
+        margin-top: $gutter;
+
+        @include media-query(large) {
+            display: none;
+        }
+
+        &:first-child {
+            margin-right: $gutter;
+        }
+
+        #{$root}__button {
+            position: initial;
+
+            &.glide__arrow--disabled {
+                opacity: 0.3;
+                pointer-events: none;
+            }
+        }
+    }
+
     &__bullet-container {
         display: flex;
         flex-direction: row;
```

Comment:
For carousel, see "Student and alumni stories" section in https://rca-development.herokuapp.com/study/programme-finder/design-products-ma/ so we've had to add the buttons for mobile/tablet view only.